hazelcast/src/main/java/com/hazelcast/query/impl/AbstractIndex.java

```diff
@@ -215,11 +215,11 @@ public PerIndexStats getPerIndexStats() {
 
     private Object extractAttributeValue(Data key, Object value) {
```

Comment:
Can we use the `entry.getMetadata()` from the callers here?

Reply:
Indexing system makes it hard to retrieve metadata from the stored record. Since indexing is done once per record, I think it does not make much difference if we omit metadata here.

Reply:
Thanks!